iocage/lib/Config/Jail/BaseConfig.py

```diff
@@ -616,22 +598,31 @@ def __setitem__(  # noqa: T400
             setter_method_name = f"_set_{key}"
             if setter_method_name in object.__dir__(self):
                 setter_method = self.__getattribute__(setter_method_name)
+                setter_kwargs = dict()
+                _spec = inspect.getfullargspec(setter_method)
+                if ("skip_on_error" in _spec) or (_spec.varkw is not None):
+                    # only pass skip_on_error when method reads it
+                    setter_kwargs["skip_on_error"] = skip_on_error
+                    # do not skip in this method when the setter takes care
+                    skip_on_error = False
```

Comment:
this whole section seems a bit over-engeineered
when we can simply demand all setters have skip on error with default of `False`

Reply:
You might be right with this. Because no setter function consumes skip_on_error at the moment, I will remove this entirely.